app/controllers/FactCheckPageReviewController.scala

```diff
@@ -69,7 +69,7 @@ class FactCheckPageReviewController @Inject() (
       .fold(
         (formWithErrors: Form[FactCheckPageReview]) => { Future.successful(BadRequest(view(processId, s"/$page", formWithErrors))) },
         result => {
-          val reviewDetail = PageReviewDetail(processId, s"/$page", Some(result.answer), Complete, updateUser = Some(s"${request.credId}:${request.name}"))
+          val reviewDetail = PageReviewDetail(processId, s"/$page", "", Some(result.answer), Complete, updateUser = Some(s"${request.credId}:${request.name}"))
```

Comment:
The re-use here with blanks args suggests we need two case classes for the different situations

Reply:
yes, I said that in the slack chat as I really didn't have time to introduce a whole new class and set of tests.  I think this is something for post accessibility audit